README.md

```diff
@@ -27,8 +27,8 @@ A TODO item should support following operations:
  - show editing history
```

Comment:
I can't comment on every line, so I write my comment here:

> A TODO item should support following operations:

Well, this is not precise techically. Because TODO item can't create new TODO item. It's better to say that `the following operations with TODO item are supported:`